tests/cppe/scf-hf/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+include(TestingMacros)
+
+add_regression_test(cppe-scf-hf "psi;cppe;addon;scf")
```

Comment:
Use commands like https://github.com/psi4/psi4/blob/master/tests/cubeprop/CMakeLists.txt here to get your `*.pot` file available at test-time.

Reply:
Addressed in 1930d51b585da776121e44a700b166c00765a99d.
Thanks!